pages/portfolio.html

```diff
@@ -4,9 +4,31 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Portfolio</title>
+  <link href="/styles/style.css" rel="stylesheet">
 </head>
+
+
+<!-- I could not for the life of me get the styling to work on this page and honestly I'm done lol -->
 <body>
-  
-</body>
-</html>
\ No newline at end of file
+    <header>
+      <nav>
+        <ul>
+          <li><a href="./about.html">About Me</a></li>
+          <li><a href="./index.html">Home</a></li>
+          <li><a href="./contact.html">Contact</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div>
```

Comment:
Are you using this div tag to apply styles? Since there aren't any ids or classes on it, I don't think it's being used for styling so you can remove it altogether since it's not providing additional structure to the page